pkg/server/plugin/upstreamauthority/spire/spire_test.go

```diff
@@ -288,13 +292,21 @@ func TestSpirePlugin_SubmitInvalidCSR(t *testing.T) {
 		csr, _, err := util.NewCSRTemplate(invalidSpiffeID)
 		require.NoError(t, err)
 
-		resp, err := m.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr})
+		stream, err := m.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr})
+		require.NoError(t, err)
+		require.NotNil(t, stream)
+
+		resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.